pkgs/development/python-modules/correctionlib/default.nix

```diff
@@ -2,6 +2,7 @@
 , buildPythonPackage
 , fetchPypi
 , cmake
+, fsspec
```

Comment:
This belongs to `python3Packages.awkward.propagatedBuildInputs` instead: https://github.com/scikit-hep/awkward/issues/2970